Changes to update FAQ and known issues section (#4260)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/customer-key-tenant-level.md

For information about rotating or rolling keys used with Customer Key, see [Roll or rotate a Customer Key or an availability key](customer-key-availability-key-roll.md). When you update the DEP to use the new version of the keys, you'll run the Set-M365DataAtRestEncryptionPolicy cmdlet as described earlier in this article.

> ## Known Issues

Reviewer: ```suggestion
## Known issues
```

> - We see that when Customer Key is enabled, creation of team from Microsoft Teams fails.

Reviewer: ```suggestion
- When Customer Key is enabled, the creation of a new team in Microsoft Teams fails.
```
